src/docs/release/breaking-changes/platform-views-using-html-slots-web.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Using HTML slots to render platform views in the web
+description: iframes in Flutter web used to reload, because of the way some DOM operations were made. A change in the way Flutter web apps render Platform Views makes them much more stable (preventing iframe reloads, and other problems with video tags or forms potentially losing their state).
+---
+
+## Summary
+
+Flutter now renders Platform Views using [slot elements][] inside of a single,
+app-wide [Shadow Root][]. Slot elements can be added/removed/moved around the
+Shadow DOM without affecting the underlying slotted content (which is rendered
+in a constant location)
+
+## Context
+
+The Flutter framework frequently tweaks its Render Tree to optimize the paint
+operations that are ultimately made per frame. In the web, these Render Tree
+changes often result in DOM operations.
+
+Flutter web used to render its Platform Views (`HtmlElementView` widgets)
+directly into its corresponding position of the DOM.
+
+Using certain DOM elements as the "target" of some DOM operations causes those
+elements to lose their internal state. In practice this means that `iframe` tags
+are going to reload, `video` players might restart, or a form being edited might
+lose the edits made to it.
+
+This change was made to:
+
+* make Platform Views behave in a stable way in Flutter web.
+* unify the way Platform Views were rendered in the web for both rendering
+backends (`html` and `canvaskit`).
+
+## Description of change
+
+A Flutter web app is now rendered inside a common Shadow Root in which slot
+elements are used to represent platform views. The actual content of each
+platform view is rendered *outside* of the Shadow Root, as a child of the Shadow
+Root host.
+
+### Before
+
+<!-- skip -->
+```html
+...
+
+<flt-glass-pane>
+  ...
+  <div id="platform-view">Contents</div> <!-- canvaskit -->
+  <!-- OR -->
+  <flt-platform-view>
+    #shadow-root
+    | <div id="platform-view">Contents</div> <!-- html -->
+  </flt-platform-view>
+  ...
+</flt-glass-pane>
+
+...
+```
+
+### After
+
+<!-- skip -->
+```html
+...
+
+<flt-glass-pane>
+  #shadow-root
+  | ...
+  | <flt-platform-view-slot>
+  |   <slot name="platform-view-1" />
+  | </flt-platform-view-slot>
+  | ...
+  <flt-platform-view slot="platform-view-1">
+    <div id="platform-view">Contents</div>
+  </flt-platform-view>
+  ...
+</flt-glass-pane>
+
+...
+```
+
+Now, when the framework needs to move DOM nodes around, it'll operate over
+`flt-platform-view-slot`s, which only contain the `slot` tag that "projects" the
+content defined in the `flt-platform-view` tags outside of the Shadow Root
+(which never move), thus preventing the reloads.
+
+From an app's perspective, this change is transparent, **however** this was
+deemed a _breaking change_ because tests that make assumptions about the DOM of
+a Flutter web app can break.
```

Comment:
```suggestion
From an app's perspective, this change is transparent. **However**, this is
considered a _breaking change_ because some tests make assumptions
about how the DOM of a Flutter web app might break.
```